docs/backup-restore.md

```diff
@@ -1,32 +1,85 @@
+
+If you want to create a transactionally-consistent cluster-wide backup, please follow the [General way to create a transactionally-consistent backup](#general-way-to-create-a-transactionally-consistent-backup) section, or 
+stop the Cassandra cluster and take the copies of all the nodes of the cluster, and start the cluster. 
+
+To minimize mistakes when doing backup operations, it is recommended to use [Cassy](https://github.com/scalar-labs/cassy).
+Cassy is also integrated with `scalar-admin` so it can issue a pause request to the application of a Cassandra cluster.
+Please see [the doc](https://github.com/scalar-labs/cassy/blob/master/docs/getting-started.md#take-cluster-wide-consistent-backups) for more details.
+
+**Cosmos DB**
+
+You must create a Cosmos DB account with a Continuous backup policy to create point-in-time restore (PITR).
+Please follow the [General way to create a transactionally-consistent backup](#general-way-to-create-a-transactionally-consistent-backup) section to create a backup.
+
+**DynamoDB**
+
+You must create tables with point-in-time recovery (PITR) and autoscaling in DynamoDB. Scalar DB Schema Loader enables PITR and autoscaling by default.
+Please follow the [General way to create a transactionally-consistent backup](#general-way-to-create-a-transactionally-consistent-backup) section to create a backup.
+
+
+## Restore Backup
+
+### JDBC databases
+
+You can restore the backup with your favorite way for JDBC databases.
+You can use `mysql` command in MySQL and `psql` command in PostgreSQL to achieve that. 
+If you use Amazon RDS (Relational Database Service) or Azure Database for MySQL/PostgreSQL, 
+you can restore to any point within the backup retention period with the automated backup feature.
+
+### Cassandra
+
+To minimize mistakes when doing restore operations, it is recommended to use [Cassy](https://github.com/scalar-labs/cassy).
+You can restore to any point using `CLUSTER-ID` and `SNAPSHOT-ID`.
+Please see [the doc](https://github.com/scalar-labs/cassy/blob/master/docs/getting-started.md#take-cluster-wide-consistent-backups) for more details.
+
+### Cosmos DB
+
+To restore a backup, you must follow the [azure official guide](https://docs.microsoft.com/en-us/azure/cosmos-db/restore-account-continuous-backup#restore-account-portal) and change the default consistency to `STRONG` after restoring the backup.
+You must use the midtime of the pause as a restore point.
+
+### DynamoDB
+
+You must restore tables one by one from the Amazon DynamoDB console using the following steps and use the midtime of the pause as a restore point
```

Comment:
`one by one` is the must? Clarify what is the must.

Reply:
Ok, I will update the details

Reply:
Thank you for your review comments.
Document updated with clarification.